dockerfiles/piraeus-server/Makefile

```diff
@@ -1,5 +1,7 @@
 PROJECT ?= piraeus-server
 REGISTRY ?= piraeusdatastore
+K8S_AWAIT_ELECTION_VERSION ?= v0.1.0
+K8S_AWAIT_ELECTION_ARCH ?= amd64
```

Comment:
I guess we will never mix `ARCH`s and we already have such an `ARCH` make variables in other places, so I guess I'd prefer `ARCH` here (and in the `Dockerfile`).